library/Icingadb/Web/Navigation/Renderer/ProblemsBadge.php

```diff
@@ -114,9 +116,28 @@ public function render(NavigationItem $item = null)
 
         $item->setCssClass('badge-nav-item icinga-module module-icingadb');
 
+        $badge = $this->createBadge();
+
+        if ($this->linkDisabled) {
+            $item->setCssClass('badge-nav-item');
```

Comment:
What is the reason for this? This unsets `icinga-module module-icingadb`. Without it the badges look differently than other badges shown in Icinga DB Web. It also hasn't an effect to the layout, so I don't see a reason for it.